scripts/shared/inlinedHostConfigs.js

```diff
@@ -16,52 +16,77 @@ module.exports = [
       'react-flight-dom-webpack/server.node',
       'react-flight-dom-webpack',
     ],
+    paths: [
+      'react-dom',
+      'react-dom/unstable-fizz',
+      'react-dom/unstable-fizz.node',
+      'react-dom/src/server/ReactDOMFizzServerNode.js', // react-dom/unstable-fizz.node
```

Comment:
These are explicitly called out so they get ignored in "dom-browser" and vice versa.